spaceship/spec/spec_helper.rb

```diff
@@ -30,10 +32,6 @@ module SpecHelper
 ENV["DELIVER_PASSWORD"] = "so_secret"
 ENV.delete("FASTLANE_USER") # just in case the dev env has it
 
-unless ENV["DEBUG"]
-  $stdout = File.open("/tmp/spaceship_tests", "w")
-end
```

Comment:
Do we now print all test results in Circle? Or did we solve it in a different way?